app/models/manageiq/providers/workflows/automation_manager/workflow_dispatcher.rb

```diff
@@ -0,0 +1,16 @@
+class ManageIQ::Providers::Workflows::AutomationManager::WorkflowDispatcher < MiqQueueWorkerBase
+  include MiqWorker::ReplicaPerWorker
```

Comment:
If we need a new worker, I'd prefer it were a single worker that doesn't have replicas, and is just one per region (which might be sort of automatic since we only have one EMS).  (I can't remember how `ReplicaPerWorker` acts compared to the other types)

This way we can keep the service account tied to a single, well defined, deployment/pod.

Reply:
`ReplicaPerWorker` just means it isn't a per-ems worker.  If there is only a single allowed per scope is separate